DFC.ServiceTaxonomy.Editor/Recipes/Alerts.Recipe.json

```diff
@@ -0,0 +1,383 @@
+{
+  "name": "",
+  "displayName": "",
+  "description": "",
+  "author": "",
+  "website": "",
+  "version": "",
+  "issetuprecipe": false,
+  "categories": [],
+  "tags": [],
+  "steps": [
+    {
+      "name": "content",
```

Comment:
We should use `ContentNoCache` (which is faster and doesn't increase memory by populating caches).